pkg/nmstatectl/nmstatectl.go

```diff
@@ -57,7 +57,12 @@ func Show(arguments ...string) (string, error) {
 }
 
 func Set(desiredState nmstatev1alpha1.State, timeout time.Duration) (string, error) {
-	return nmstatectlWithInput([]string{"set", "--no-commit", "--timeout", strconv.Itoa(int(timeout.Seconds()))}, string(desiredState.Raw))
+	var setDoneCh = make(chan struct{})
```

Comment:
I mean to SetWithUnavailabeLinkWorkround(... at unavailable_link_workaround.go and there
do the thing and at the end nmstatectl.Set

and at client.go just replace nmstatectl.Set for SetWIthUnabl... so client.go is just one liner change and all the integration happends a new file we can easily delete.

Reply:
The bug is only related to nmstatectl execution, so I moved it here.

I don't see how introducing wrappers is easier to revert than three short lines. Both will be just `git revert` anyway.

Reply:
fair enough